lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -141,6 +163,7 @@ def _span(name: str) -> Generator[Optional[Span], Any, Any]:
         with tracer.start_as_current_span(name) as span:
             yield cast(Span, span)
     else:
+        logger.debug(f"no tracer found: {_get_tracer()}")
```

Comment:
```suggestion
        logger.debug(f"tracer not found")
```

it's probably going to be None in that case, no?

Reply:
Correct. Looks much better, changed.